### DISCUSSION TITLE
Template repository analytics

### DISCUSSION THREAD

**Discussion by bethac07** [community/community#20067] [Other Feature Feedback, Questions, & Ideas]
Briefly, it would be great to be able to track "children" of template repositories in the same way you can track stars, forks, etc; our team uses template repositories a lot, and it's great, but we essentially lose the ability to track how many people are using our tools in a way we don't with ie forks. This information is kept at SOME level, since in a "child" repository you can see which template it was made from, but there doesn't seem to be a way to find out the information in the other direction. Is this on the roadmap? Thanks!

It looks like others are [interested](https://github.community/t/is-there-a-way-to-find-repositories-generated-from-my-template-repository/122838) in this [in general](https://stackoverflow.com/questions/62943177/how-to-see-how-many-repos-created-from-my-repo-template-github/63910250#63910250), but I could not find an actual request here upon searching - apologies if I missed it!

**Comment by critesjosh** [comment]
I am interested in a feature like this as well

**Comment by sergiinvestor** [comment]
The idea of tracking 'children' of template repositories is indeed valuable for understanding the reach and utilization of shared tools. In my experience, combining GitHub's native features with external analytics tools can provide deeper insights. For instance, tools like [Gitlights](https://github.com/marketplace/gitlights-analytics) offer comprehensive analytics on repository activity, which can help in understanding how templates are being utilized across various projects. Here's an example of how Gitlights visualizes repository activity:

<img width="1259" alt="Captura de pantalla 2024-12-17 a las 21 34 39" src="https://github.com/user-attachments/assets/ac74dd50-4c65-4bf8-88a8-4651137331e6" />


Integrating such tools can complement GitHub's functionalities and provide maintainers with a broader perspective on their repositories' impact.
